### ISSUE TITLE
I sent −0.50468561 ETH ≈ $456.74 from coinbase

### ISSUE THREAD

**Issue by Edgardoga** [ethereum/mist#3659]
- [ ] I've asked for help in the [Mist Gitter](http://gitter.im/ethereum/mist) before filing this issue.

```
Version: `0.0.0`
OS & Version: windows/linux/osx
Node version: `geth 0.0.0` 
Number of blocks synchronized: 0
```

**Comment by simon66** [comment]
@Edgardoga @evertonfraga 
I'm not sure I understand your question. So you bought `0.50468561 ETH` from Coinbase and sent that full amount on another Ethereum address?

**Comment by Edgardoga** [comment]
Yes Sir, the thing is that I can't open ethereum wallet which I previously downloaded into my computer, so, I know it is there but I can't get access to it or either return it to my coinbase..., basically I did something so stupid and it's because my inexperience that I have to pay, I have probably lost that...thank you for following on this issue,

Regards.

________________________________
